src/libraries/Common/src/System/Security/Cryptography/Asn1/DirectoryStringAsn.xml.cs

```diff
@@ -8,6 +8,7 @@
 using System.Security.Cryptography;
 using System.Security.Cryptography.Asn1;
 
+#nullable enable
```

Comment:
These .xml.cs files are generated by a tool.  If we're changing them, we should also update the templates that's used to create them.  Otherwise, we'll lose these changes the next time someone runs that tool.

Reply:
I didn't test this (not sure how to run tool), but pretty sure this change in `asn.xslt`  would work as needed https://github.com/dotnet/runtime/pull/1535/files#diff-142d5cdb81d309fff608037fe74339b1R145

Reply:
I have some changes to the generator that I'm playing with, so it'll get run on everything soon enough.  The changes look OK as-is, there'll just be more regenerated code later.

Reply:
Great, thanks!